speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionMaker.inl

```diff
@@ -176,3 +179,15 @@ Recorder* TEMP_DECISIONMAKER::GetRecorder()
 {
     return m_recorder;
 }
+
+template <typename SocketBlackBox, typename SDAConfig, typename FileDataStorage, typename SQLDatabaseStorage, typename Recorder>
+DecisionTuple TEMP_DECISIONMAKER::GetDecisions()
```

Comment:
Missing documentation, I also see that documentation misses on the 4 functions above, could you also add that? Appereantly it didn't get noticed in those PRs.